ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -101,9 +104,16 @@ public void createTopic(
     newTopic.configs(toStringConfigs(configs));
 
     try {
-      LOG.info("Creating topic '{}'", topic);
+      LOG.info(String.format("Creating topic '{}' %s",
+          topic,
+          (createOptions.shouldValidateOnly()) ? "(ONLY VALIDATE)" : ""
+      ));
+
       ExecutorUtil.executeWithRetries(
-          () -> adminClient.createTopics(Collections.singleton(newTopic)).all().get(),
+          () -> adminClient.createTopics(
```

Comment:
I think the retry execution also works with network issues when attempting to create the topic. Andy told me about this when I attempted to remove the retry when deleting topics. Otherwise, I'm not sure why KSQL needs to retry for a createTopic without validation too.

Reply:
Not sure I follow - what I meant is that we should not retry if the error thrown is `TopicAuthorizationException` as opposed to (for example `IOException`).

Reply:
Got it. The executeWithRetries has this parameter `ExecutorUtil.RetryBehaviour.ON_RETRYABLE` which only retries in when the thrown exception is a `RetriableException`. A `TopicAuthorizationException` is not a `RetriableException`, so there are no retries in this case.

Reply:
nice! let's add a test to make sure it only makes one external call and then i'm good to go!

Reply:
I don't think we need a special test for this. There are enough tests on `ExecutorUtilTest` that verify RETRYABLE exceptions are retried. 

If for some reason `TopicAuthorizationException` is wrapped in a Retryable exception, then that would mean Kafka decided that the exception must be retried for some reason. Or a bug from them, but why would we want to verify that other APIs do not have bugs?

Reply:
Fair enough.